src/diffusers/__init__.py

```diff
@@ -88,26 +100,43 @@
         VersatileDiffusionTextToImagePipeline,
         VQDiffusionPipeline,
     )
-else:
-    from .utils.dummy_torch_and_transformers_objects import *  # noqa F403
 
-if is_torch_available() and is_transformers_available() and is_k_diffusion_available():
-    from .pipelines import StableDiffusionKDiffusionPipeline
+try:
+    if not (is_torch_available() and is_transformers_available() and is_k_diffusion_available()):
+        raise OptionalDependencyNotAvailable()
+except OptionalDependencyNotAvailable:
+    from .utils.dummy_torch_and_transformers_and_onnx_objects import *  # noqa F403
```

Comment:
this seems incorrect 
```suggestion
    from .utils.dummy_torch_and_transformers_and_k_diffusion_objects import *  # noqa F403
```